Lesson02/StartOfLesson/ToDo/Controllers/StatusController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ToDoApp.Models;
+using ToDoApp.Services;
+
+namespace ToDoApp.Controllers
+{
+    public class StatusController : Controller
+    {
+        public ActionResult Index()
+        {
+            return View(Repository.StatusList);
+        }
+        public ActionResult Create()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Create(Status collection)
+        {
+            try
+            {
+                return RedirectToAction(nameof(Index));
+            }
+            catch
+            {
+                return View();
+            }
+        }
+        public ActionResult Edit(Status Collection)
```

Comment:
This is not quite right. Since this is a http GET handler or to put it another way, this is the method that gets called when we GET a page in the browser, we want the information from the URL such as the ID of the record we want to edit (not the whole record)
```suggestion
        public ActionResult Edit(int id)
```